app/com/yetu/oauth2provider/views/registration.scala.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+
+<html>
+	<head>
+			<title>Registration</title>
+			<link rel="stylesheet" media="screen" href="@controllers.routes.Assets.at("stylesheets/setup.css")">
+			<link rel="stylesheet" media="screen" href="@controllers.routes.Assets.at("stylesheets/registration.css")">
+			<link rel="shortcut icon" type="image/png" href="">
+			<meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
+	</head>
+	<body>
+		<header>
+			<h1>@Messages("setup.headline")</h1>
+			<h2>@Messages("setup.subheadline")</h2>
+		</header>
+		<div class="steps">
+			<ul class="steplist">
+				<li class="step active">@Messages("setup.steps.one")</li>
+				<li class="step">@Messages("setup.steps.two")</li>
+				<li class="step">@Messages("setup.steps.three")</li>
```

Comment:
I think we should help to split this stuff to components, it would be good if @ElliDy or @meedheym would be involved.